qiskit/circuit/gate.py

```diff
@@ -219,3 +219,13 @@ def broadcast_arguments(self, qargs, cargs):
             return Gate._broadcast_3_or_more_args(qargs)
         else:
             raise CircuitError('This gate cannot handle %i arguments' % len(qargs))
+
+    def normalize_parameter(self, parameter):
+        # examples:
+        #  u3(0.1, 0.2, 0.3)
+        #  u2(pi/2, sin(pi/4))
+        if isinstance(parameter, (int, float)):
+            return parameter
+
+        else:
+            return super().normalize_parameter(parameter)
```

Comment:
I'm confused how is this different from the parent `Instruction` class? If parameter is a list str complex or parameter expression this just will return parameter too

Reply:
sorry.. I'm still merging this.

Reply:
You are right. Removed in c5b7fe819.

Reply:
I fully changed it... have another look!